instrumentation/amazing_gem/Gemfile

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
```

Comment:
The [preferred copyright notice](https://github.com/cncf/foundation/blob/master/copyright-notices.md) is:
```ruby
# Copyright The OpenTelemetry Authors
```
I.e. we shouldn't include the year.

Reply:
Should the license be updated as well? 

https://github.com/open-telemetry/opentelemetry-ruby/pull/497/files#diff-83b5e236433acba18ca2306b043b5c196bf7bf2c5eab692dbd90eabc0c20d4f1R189

Reply:
Good catch! Yes, please.